docs/configuration.asciidoc

```diff
@@ -512,6 +512,23 @@ Set it to `0` to disable stack trace collection for all spans.
 
 It has to be provided in *<<config-format-duration, duration format>>*.
 
+[float]
+[[config-ssl-ca-cert]]
+==== `ssl_ca_cert`
+
+[options="header"]
+|============
+| Environment                | `Config` key    | Default | Example
+| `ELASTIC_APM_SSL_CA_CERT`  | `ssl_ca_cert`   | `nil`   | /path/to/ca.pem
+|============
+
+This parameter should contain the path to a .pem formatted CA certificate. This
+allows verification of self-signed certificates instead of disabling server
+certificate verification.
+
+The path provided must be accessible to the agent for HTTP.rb to load the
```

Comment:
I'd just leave out the last part, `  for HTTP.rb to load the certificate.` Implementation detail 😉

Reply:
One of the things I found frustrating about the Elastic documentation is that there are a lot of details that are hinted at but not defined. I added this as a helpful note for those implementing this. In general, I’m in favor of making things easier for people.

I agree with you on this case, though, and am happy to remove.

Reply:
I'm sorry to hear. I want the docs of the agent to be as good as possible but it's never easy to walk the line between too much and too little. If you remember or find any places you think are lacking please let me know where it is and I'll see what I can do!

Reply:
Looking again at this, I totally agree with you on the HTTP.rb point. It should not be in the docs and I will remove it.

Reply:
Done.